knora-ontologies/knora-base.ttl

```diff
@@ -2703,11 +2920,99 @@ rdfs:label rdfs:subPropertyOf :objectCannotBeMarkedAsDeleted .
                                                "Representation (Text)"@en ,
                                                "Répresentation (Texte)"@fr ,
                                                "Rappresentazione (testo)"@it ;
-                    
+
                     rdfs:comment "A resource containing text files"@en .
 
 
 
+###  http://www.knora.org/ontology/knora-base#Person
+
+:Person rdf:type owl:Class ;
+
+        rdfs:subClassOf :Resource, foaf:Person,
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasFamilyName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasGivenName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasIAFIdentifier ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasBirthDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasDeathDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ];
+
+        rdfs:label "Person"@de ,
+                   "Person"@en ,
+                   "Personne"@fr ,
+                   "Persona"@it ;
+
+        rdfs:comment "A resource representing a person"@en .
+
+
+###  http://www.knora.org/ontology/knora-base#Publication
+
+:Publication rdf:type owl:Class ;
```

Comment:
I understand wanting to use existing vocabularsies, FOAF was designed for personal profile pages, not for humanities research. A `foaf:person` has a birthday (month and day) but no year of birth and no date of death. It doesn't have a way to specify that a person is known by multiple names, and that one of those names is canonical. It actually seems to me that there isn't anything in `foaf:Person` that can be used as-is in the humanities.

And some of the things you want to add to `knora-base` seem rather BEOL-specific. Why create a class called `Publication` instead of something more general like `Work`? Not everything that can be cited in a bibliography is a publication.

My guess is that it will take time to answer questions like this, and it will require input from more than one project, not just BEOL. Meanwhile, I think it would make sense to create a separate bibliography ontology that could be considered less stable than `knora-base`.